catalog/package.json

```diff
@@ -40,7 +40,7 @@
     "pretest": "npm run test:clean && npm run lint",
     "test:clean": "rimraf ./coverage",
     "test": "cross-env NODE_ENV=test jest --coverage",
-    "test:watch": "cross-env NODE_ENV=test jest --watchAll",
+    "test:watch": "cross-env NODE_ENV=test jest --watch",
```

Comment:
why is this required?

Reply:
It's not required. I don't see reasons, why we need `--watchAll` if `--watch` is enough
If there are reasons, please elaborate.